src/Microsoft.DotNet.Interactive/Parsing/SubmissionParser.cs

```diff
@@ -481,10 +485,17 @@ private PolyglotParserConfiguration GetParserConfiguration()
 
         if (expressionType is "input" or "password")
         {
-            var parametersNode = expressionNode.ChildNodes.OfType<DirectiveExpressionParametersNode>().SingleOrDefault();
+            if (command is SubmitCode)
```

Comment:
Is this always going to be a `SubmitCode`? If yes, should this throw an exception if some other command is present?

Reply:
Submission splitting happens for many different command types, including language service commands. We don't want the non-`SubmitCode` commands to trigger user prompts though.